Library/Homebrew/cmd/pull.rb

```diff
@@ -190,6 +190,10 @@ def pull
           safe_system 'brew', install, '--debug', f.name
         end
       end
+
+      unless ARGV.include? "--no-repair"
```

Comment:
Yup, works :+1:. Does the repair for taps, doesn't do the repair for Homebrew/homebrew PRs. Cool. Pushing shortly.